opennms-web-api/src/main/java/org/opennms/web/svclayer/support/DefaultAdminApplicationService.java

```diff
@@ -237,6 +254,7 @@ public void performEditServices(String applicationIdString, String editAction, S
             }
 
             m_applicationDao.save(application);
+            sentEvent(application, EventConstants.APPLICATION_CHANGED_EVENT_UEI);
```

Comment:
Same must be done in the performEditLocations() method, since the application is also saved there.

Reply:
:+1: done.